libopae++/include/opaec++/pvalue.h

```diff
@@ -52,6 +52,10 @@ struct guid_t
         return nullptr;
     }
 
+    const uint8_t* get() const {
+        return data_.data();
+    }
+
```

Comment:
Just curious - what happens to the guid the returned pointer is pointing to if the properties object gets destroyed?

Reply:
Hmm... It becomes a dangling pointer

Reply:
Uh, okay. That documented anywhere? Or is that implicit?

I just wonder if someone wouldn't get a pointer to the guid and discard the original...

Reply:
Not documented yet... 😄